cmd/kogito/command/common/channel_flag.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2020 Red Hat, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package common
+
+import (
+	"fmt"
+	"github.com/kiegroup/kogito-cloud-operator/cmd/kogito/command/shared"
+	"github.com/spf13/cobra"
+)
+
+type ChannelFlags struct {
```

Comment:
Instead of `ChannelFlags` could you please rename it to `OperatorFlags`? This way we can handle future flags to this structure.

Reply:
done